### PR TITLE
	new file:   packages/raknet/src/proto/data/NewIncomingConnection.ts

### PR DESCRIPTION
**Pull request by DKMtomy**
i should have implemented the packet correctly

### CODE REVIEW

File: packages/raknet/src/proto/data/NewIncomingConnection.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { Long, Bool, Short } from "@serenityjs/binarystream";
+
+import { Proto, Serialize } from "../../decorators";
+import { Packet } from "../../enums";
+import { Address, Magic } from "../types";
+
+import { BasePacket } from "./base";
+import { serialize } from "v8";
```
Comment (PMK744) [inline packages/raknet/src/proto/data/NewIncomingConnection.ts:8]: This should be removed

File: packages/raknet/src/proto/data/NewIncomingConnection.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+/**
+ * Represents an open connection reply 1 packet.
+ */
+@Proto(Packet.NewIncomingConnection)
+class NewIncomingConnection extends BasePacket {
+  /**
+   * the server adress of the reply.
+   */
+  @serialize(Address) public serverAddress!: string;
```
Comment (PMK744) [inline packages/raknet/src/proto/data/NewIncomingConnection.ts:18]: Needs to be Serialize

File: packages/raknet/src/proto/data/NewIncomingConnection.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  /**
+   * unknown what this is used for.
+   */
+  @serialize(Address) public internalAddress!: string;
```
Comment (PMK744) [inline packages/raknet/src/proto/data/NewIncomingConnection.ts:23]: Needs to be Serialize

**Comment by PMK744** [comment]
File names also need to be in kebab space, (new-incoming-connection.ts)

**Comment by DKMtomy** [comment]
sorry my bad, should be fixed now

**Pull request merged**